app/assets/javascripts/utils/api.js

```diff
@@ -66,6 +66,23 @@ const API = {
     });
   },
 
+  fetchMoreRevisions(courseId, limit) {
```

Comment:
Similarly, here you can probably just adjust fetchRevisions to take a limit, and then stick with a single fetch method here.